include/amici/misc.h

```diff
@@ -11,6 +11,17 @@
 #include <gsl/gsl-lite.hpp>
 
 namespace amici {
+    
+/**
+ * @brief creates a slice from existing data
+ *
+ * @param data to be sliced
+ * @param index slice index
+ * @param size slice size
+ */
+ 
+ gsl::span<realtype> slice(std::vector<realtype> data, const int index,
```

Comment:
See also gsl-lite.hpp: `span subspan( index_type offset, index_type count )`

Reply:
subspan does not multiply the offset with size.